### ISSUE TITLE
Implementation of eqn. 15

### ISSUE THREAD

**Issue by limchaos** [Haoqing-Wang/InfoCL#6]
Hi, 

Thanks for the interesting work. The implementation _loss = criterion(z_i, z_j) - args.lamb * MI_estimitor_ seems not aligned with eqn. 15 (where you minus InfoNCE in the implementation but eqn. 15 is adding InfoNCE). 

_# InfoNCE Lower bound
def InfoNCE(mu, z):
    mu = mu.unsqueeze(0)
    z = z.unsqueeze(1)
    score = -((z-mu)**2).sum(-1)/20.  # 80 for ImageNet
    lower_bound = -score.logsumexp(dim=1).mean()
    return lower_bound_

Maybe I miss something, can you explain a bit, thanks!

**Comment by limchaos** [comment]
Sorry, solved, that is a maximization
